dspace-api/src/main/java/org/dspace/content/service/ItemService.java

```diff
@@ -768,6 +769,29 @@ public Iterator<Item> findByLastModifiedSince(Context context, Date last)
      */
     int countWithdrawnItems(Context context) throws SQLException;
 
+    /**
+     * finds all items for which the current user has editing rights
+     *
+     * @param context DSpace context object
+     * @param offset  page offset
+     * @param limit   page size limit
+     * @return list of items for which the current user has editing rights
+     * @throws SQLException
+     * @throws SearchServiceException
+     */
+    public List<Item> findItemsWithEdit(Context context, String q, int offset, int limit)
+        throws SQLException, SearchServiceException;
+
+    /**
+     * counts all items for which the current user has editing rights
+     *
+     * @param context DSpace context object
+     * @return list of items for which the current user has editing rights
+     * @throws SQLException
+     * @throws SearchServiceException
+     */
+    public int countItemsWithEdit(Context context, String q) throws SQLException, SearchServiceException;
```

Comment:
Same here, update the JavaDocs to include `String q` please

Reply:
Added the doc.